docs/find-duplicate-skus/script.liquid

```diff
@@ -0,0 +1,103 @@
+{% assign mappings = hash %}
```

Comment:
Always err on the side of _more_ descriptive, with variable names. What are we mapping here?

I tend to name hashes using something like "x_by_y", where "x" describes the values and "y" describes the keys. In this case, that would be "variant_ids_by_sku". You're camelCasing, so that'd be "variantIdsBySku", here. :)